reference/configuration/framework.rst

```diff
@@ -2499,6 +2499,67 @@ enabled
 validation
 ~~~~~~~~~~
 
+.. _reference-validation-auto-mapping:
+
+auto_mapping
+............
+
+**type**: ``array`` **default**: ``[]``
+
+Defines the Doctrine entities that will be introspected to add
+:ref:`automatic validation constraints <automatic_object_validation>` to them:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        framework:
+            validation:
+                auto_mapping:
+                    # an empty array means that all entities that belong to that
+                    # namespace will add automatic validation
+                    'App\Entity\': []
+                    'Foo\': ['Foo\Some\Entity', 'Foo\Another\Entity']
+
+    .. code-block:: xml
+
+        <!-- config/packages/framework.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:validation>
+                    <framework:auto-mapping>
+                        <framework:namespace>App\Entity\</framework:namespace>
+                        <framework:service/>
+
+                        <framework:namespace>Foo\</framework:namespace>
+                        <framework:service>Foo\Some\Entity</framework:service>
+                        <framework:service>Foo\Another\Entity</framework:service>
```

Comment:
I'm not very experienced with config XML, but the namespace should be an attribute as it seems from https://github.com/symfony/symfony/blob/6c85287f18120b6a00014eec806a6427bbb6e080/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L1030

Should it be like this?
```xml
<framework:service namespace="App\Entity\"/>
<framework:service namespace="Foo\">Foo\Some\Entity</framework:service>
<framework:service namespace="Foo\">Foo\Another\Entity</framework:service>
```